Procfile

```diff
@@ -1 +1 @@
-web: sh -c 'cd server && npm i && npm run build && node --max_old_space_size=400 --gc_interval=100 --harmony dist/app.js'
```

Comment:
I think this can go entirely. @metasoarous ?

Reply:
I assume this was a heroku requirement

Reply:
We don't use the procfile anymore, but rather the heroku.yml, so procfile can go away

Reply:
removed